sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentAnalysisModels/OperationsLiveTests.cs

```diff
@@ -135,10 +135,11 @@ public async Task ClassifyDocumentOperationCanPollFromNewObject()
             await sameOperation.WaitForCompletionAsync();
 
             Assert.IsTrue(sameOperation.HasValue);
-            Assert.AreEqual(4, sameOperation.Value.Pages);
+            Assert.AreEqual(4, sameOperation.Value.Pages.Count);
         }
 
         [RecordedTest]
+        [ServiceVersion(Min = DocumentAnalysisClientOptions.ServiceVersion.V2023_02_28_Preview)]
```

Comment:
Some tests meant for the new service features were not marked with the `ServiceVersion` attribute.